google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java

```diff
@@ -177,13 +178,24 @@ public void close() {
    * <p>Sample code:
    *
    * <pre>{@code
+   * // A table with a single column family, which retains only the latest value.
    * Table table = client.createTable(
    *   CreateTableRequest.of("my-table")
-   *     .addFamily("cf2", GCRULES.maxVersions(1)
+   *     .addFamily("cf2", GCRULES.maxVersions(1))
+   * );
```

Comment:
There seems to be an extra closing paren

Reply:
This parenthesis is to close the createTable, Please advise me if this needs to be inlined with `#addFamily`

Reply:
my mistake